settings.gradle.kts

```diff
@@ -26,10 +26,13 @@ include(":moshi")
 include(":moshi:japicmp")
 include(":moshi:records-tests")
 include(":adapters")
+project(":adapters").name = "moshi-adapters"
```

Comment:
In a follow-up I’d like to move the files around so we don’t need this renaming hack.